### PR TITLE
Deploy new XUI Slack Bot

### PR DESCRIPTION
**Pull request by danlysiak**
### Change description ###
Deploy new XUI Slack Bot

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: xui-slack-help-bot
+  namespace: xui-slack-help-bot
+spec:
+  releaseName: xui-slack-help-bot
+  values:
+    replicas: 1
+    image: hmctspublic.azurecr.io/xui/slack-help-bot:prod-tbc #{"$imagepolicy": "flux-system:xui-slack-help-bot"}
```
Comment (danlysiak) [inline apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml:10]: TODO: Update

File: apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    environment:
+      # SLACK_REPORT_CHANNEL: platops-help
+      # SLACK_REPORT_CHANNEL_ID: C8SR5CAMU
+      # JIRA_START_TRANSITION_ID: 181
+      # JIRA_DONE_TRANSITION_ID: 201
+      # JIRA_PROJECT: DTSPO
+      # JIRA_ISSUE_TYPE_ID: 17100
+      # JIRA_ISSUE_TYPE_NAME: "BAU Task"
```
Comment (danlysiak) [inline apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml:18]: TODO: Update

File: apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+metadata:
+  name: xui-slack-help-bot
+  namespace: xui-slack-help-bot
+spec:
+  releaseName: xui-slack-help-bot
+  values:
+    replicas: 1
+    image: hmctspublic.azurecr.io/xui/slack-help-bot:prod-ee7658a-20230315123456 #{"$imagepolicy": "flux-system:xui-slack-help-bot"}
+    environment:
+      SLACK_REPORT_CHANNEL: exui-bot-temp
```
Comment (danlysiak) [inline apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml:12]: Will change in a future PR - using for initial testing.

File: apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      JIRA_PROJECT: EXUI
+      JIRA_ISSUE_TYPE_ID: 10900
+      JIRA_ISSUE_TYPE_NAME: "Bug"
+    aadIdentityName: xui-slack-help-bot
+    keyVaults:
+      cftptl-intsvc:
+        excludeEnvironmentSuffix: true
+        secrets:
+          - xui-slack-bot-token
+          - xui-slack-app-token
```
Comment (danlysiak) [inline apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml:23]: Will merge PR after adding these to vault.

File: apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+          - jira-username
+          - jira-api-token
+  chart:
+    spec:
+      chart: nodejs
+      version: 2.4.8
```
Comment (timja) [inline apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml:29]: use the latest version please
```suggestion
      version: 2.4.14
```

Comment (danlysiak) [reply apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml]: Serves me right for copy/pasting.. done!

**Pull request merged**